ml/ad_charts.cc

```diff
@@ -327,7 +327,7 @@ void ml_update_host_and_detection_rate_charts(ml_host_t *host, collected_number
                     "anomaly_detection", // family
                     "anomaly_detection.detector_events", // ctx
                     "Anomaly detection events", // title
-                    "percentage", // units
+                    "events", // units
```

Comment:
I think this is still wrong and that is why:

 - the chart contains 2 dimensions: `above_threshold` and `new_anomaly_event`.
 - both [these values are boolean](https://github.com/netdata/netdata/blob/0051e8167c25ebc9e49e7733557503addc8b6832/ml/ad_charts.cc#L374-L375).

I guess this should be split into 2 status charts.

Reply:
yeah - that will mean FE work needed on anomalies tab then since this chart is used there.

i think `boolean` works for now as a improvement, similar to [this](https://github.com/netdata/netdata/blob/0051e8167c25ebc9e49e7733557503addc8b6832/ml/ad_charts.cc#L204)